### ISSUE TITLE
Audible re-evaluation cues

### ISSUE THREAD

**Issue by amyjko** [wordplaydev/wordplay#537]
### What are you trying to do that you can't?

When programs re-evaluate, the editor throbs streams that are updates visually, but that cue is not available for people relying on audio cutes.

### What is your idea?

Mirror re-evaluation in the auditory channel, like the audio cues in VS Code:

https://github.com/microsoft/vscode/issues/147386

This should be provided, but also taught in some way.

**Comment by VarunP3000** [comment]
@amyjko I would like to help develop a feature that creates audio updates anytime the program is re-evaluated. I read the discussion about mirror re-evaluation and JAWS in the audio cues in VSCODE and gained some insight. I would also like to learn more about the issue before I get started, in case I get assigned.

**Comment by amyjko** [comment]
@VarunP3000 verbally opted out of this for now.
